### DISCUSSION TITLE
Setting up feitian ePass 2003 issues

### DISCUSSION THREAD

**Discussion by Jakuje** [OpenSC/OpenSC#2835] [Q&A] [answered]
I recently received the ePass 2003, but I have been struggling how to set it up to be able to do something useful with this token.

There are only very outdated guides  (dating 10 years back and simple `pkcs15init -E` and `pkcs15init -C` fails for me:
```
$ pkcs15-init --create-pkcs15 --profile pkcs15+onepin --label "myToken" --pin 123456 --puk 12345678 --reader "Feitian ePass2003 00 00" --use-default-transport-key
Failed to create PKCS #15 meta structure: Not allowed
```
It would be great to have the setup process and steps documented somwhere, ideally in wiki as we already have a page for this token: https://github.com/OpenSC/Wiki/blob/master/Feitian-ePass2003.textile

I know @popovec was working with this token recently so you can be some help. Also @FeitianSmartcardReader might be able to point out into the right direction.

**Comment by popovec** [comment, marked as answer]
I also encountered the problem with "Failed to create PKCS#15 meta structure: Not allowed", so far it still helped me to erase the token: `pkcs15-init -E -T` and then to initialize the token using `pkcs15-init -C -T -p pkcs15+onepin --pin 1234 --puk 123456`. 

Try looking at the log to see if the `pkcs15-init -E` operation went correctly (this operation installs a new PIN, then performs a VERIFY operation with it, and then deletes the MF).

**Reply by Jakuje** [reply]
Interesting. Now with erase and create (both with `-T` default transport keys) worked. Thanks for the pointer!

**Reply by Jakuje** [reply]
Added the steps I used (and that worked for me): https://github.com/OpenSC/Wiki/pull/2
